hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamManagerImpl.java

```diff
@@ -130,6 +131,7 @@ public class BlockStreamManagerImpl implements BlockStreamManager {
     private Bytes lastBlockHash;
     private Instant blockTimestamp;
     private Instant consensusTimeLastRound;
+    private Instant lastExecutionTime;
```

Comment:
We might keep this as a PBJ `Timestamp` and only convert to `Instant` when we need to invoke an `itemSpec()` (since that will happen very rarely compared to the number of calls to `writeItem(BlockItem)`).

Reply:
Ok, makes sense 👍🏻